sydent/util/emailutils.py

```diff
@@ -73,7 +74,24 @@ def sendEmail(
     # conflicting with it.
     allSubstitutions["multipart_boundary"] = generateAlphanumericTokenOfLength(32)
 
-    mailString = open(templateFile).read() % allSubstitutions
+    # Render template with Jinja if using Jinja template
+    root_template_path = sydent.cfg.get("general", "templates.path")
+    env = Environment(
+        loader=FileSystemLoader(root_template_path),
+        autoescape=select_autoescape(["html", "xml"]),
+    )
```

Comment:
ah, the problem here is that we only want to create one Environment (forever) and keep reusing it.

I don't think the autoescape functionality is right here: `select_autoescape` says it will choose based on the file extension. Our e-mail files end with the `.eml` extension, so it will decide *not* to auto-escape.

I think you're better off:
* making it auto-escape always (unconditionally)
* using `| safe` syntax (described somewhere before in this PR's comments) for places where escaping is not necessary (i.e. in plain-text fragments of the e-mail)